pymodbus/framer/rtu_framer.py

```diff
@@ -236,3 +304,8 @@ def recvPacket(self, size):
         result = self.client.recv(size)
         self.client.last_frame_end = round(time.time(), 6)
         return result
+
+    def __del__(self):
```

Comment:
The belongs in drain(True)

We do not do cleanup in __del__ anywhere in the code, especially because the client object might be reused after a close().